src/v2/Artsy/Router/Utils/shouldUpdateScroll.tsx

```
@@ -7,7 +7,7 @@
 export function shouldUpdateScroll(_prevRenderArgs, { routes }) {
   try {
     // If true, don't reset scroll position on route change
-    if (routes.some(route => route.ignoreScrollBehavior)) {
+    if (routes?.some(route => route.ignoreScrollBehavior)) {
```

This has masked errors in the past since it sits so low in the stack